tests/migrations/models.py

```diff
@@ -72,3 +72,11 @@ class FoodManager(BaseFoodManager.from_queryset(FoodQuerySet)):
 
 class NoMigrationFoodManager(BaseFoodManager.from_queryset(FoodQuerySet)):
     pass
+
+
+class ExoPlanet(models.Model):
+    name = models.CharField(max_length=255)
+    inhabitable = models.BooleanField(default=False)
+
+    class Meta:
+        app_label = 'exoplanets'
```

Comment:
I don't understand why an explicit `app_label` is specified here. It looks strange to me that this model is supposed to belong to the `exoplanets` application but is created as part of the `migrations` apps migrations.